### DISCUSSION TITLE
Implement hosts for Dockerfile build

### DISCUSSION THREAD

**Discussion by jlongtine** [dagger/dagger#12526] [Ideas]
This allows a user to add hosts to the Dockerfile build

```graphql
input Host {
  host: String!
  ip: String!
}

type Directory {
  dockerBuild(dockerfile: String, platform: Platform, hosts: [Host!]): Container!
}
```

**Comment by grouville** [comment]
@samalba, as it is the last issue on https://github.com/dagger/dagger/issues/4241. How do you feel about the priority of this one ? I recall that @shykes had some reserves around the design of the host /engine interactions ?
